beekeeper-path-scheduler-apiary/src/test/java/com/expediagroup/beekeeper/scheduler/context/CommonBeansTest.java

```diff
@@ -0,0 +1,80 @@
+package com.expediagroup.beekeeper.scheduler.context;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.EnumMap;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.expedia.apiary.extensions.receiver.common.messaging.MessageReader;
+import com.expedia.apiary.extensions.receiver.sqs.messaging.SqsMessageReader;
+
+import com.expediagroup.beekeeper.core.model.LifecycleEventType;
+import com.expediagroup.beekeeper.scheduler.apiary.context.CommonBeans;
+import com.expediagroup.beekeeper.scheduler.apiary.filter.FilterType;
+import com.expediagroup.beekeeper.scheduler.apiary.filter.ListenerEventFilter;
+import com.expediagroup.beekeeper.scheduler.apiary.messaging.BeekeeperEventReader;
+import com.expediagroup.beekeeper.scheduler.apiary.messaging.RetryingMessageReader;
+import com.expediagroup.beekeeper.scheduler.service.SchedulerService;
+
+@ExtendWith(MockitoExtension.class)
+public class CommonBeansTest {
+
+  private static final String AWS_S3_ENDPOINT_PROPERTY = "aws.s3.endpoint";
+  private static final String AWS_REGION_PROPERTY = "aws.region";
+  private static final String REGION = "us-west-2";
+  private static final String AWS_ENDPOINT = String.join(".", "s3", REGION, "amazonaws.com");
+  private static final String ENDPOINT = "endpoint";
+  private static final String BUCKET = "bucket";
+  private static final String KEY = "key";
+  private final CommonBeans commonBeans = new CommonBeans();
+  @Mock private MessageReader messageReader;
+
+  @AfterAll
+  static void teardown() {
+    System.clearProperty(AWS_REGION_PROPERTY);
+    System.clearProperty(AWS_S3_ENDPOINT_PROPERTY);
+  }
+
+  @BeforeEach
+  void setUp() {
```

Comment:
Can you swap `setUp()` and `tearDown()` method order please, it just reads a bit easier.